<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>dspItemCostsByClassCode</class>
 <widget class="QWidget" name="dspItemCostsByClassCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item Costs by Class Code</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ParameterGroup" name="_classCode">
       <property name="title">
        <string/>
       </property>
       <property name="fixedSize">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="XCheckBox" name="_onlyShowZero">
     <property name="text">
      <string>Only Show &amp;Zero Cost Items</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="XCheckBox" name="_onlyShowDiff">
     <property name="text">
      <string>Only Show Items where Actual and Standard are different</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="XCheckBox" name="_showInactive">
     <property name="text">
      <string>Show Inactive Items</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>ParameterGroup</class>
   <extends>QGroupBox</extends>
   <header>parametergroup.h</header>
  </customwidget>
  <customwidget>
   <class>XCheckBox</class>
   <extends>QCheckBox</extends>
   <header>xcheckbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_classCode</tabstop>
  <tabstop>_onlyShowZero</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
